spec/core_functions/color/is_legacy.hrx

```diff
@@ -0,0 +1,284 @@
+a {
+  b: true;
+}
+
+<===>
+================================================================================
+<===> hex/input.scss
+@use "sass:color";
+a {b: color.is-legacy(#f2ece4)}
+
+<===> hex/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> named/input.scss
+@use "sass:color";
+a {b: color.is-legacy(midnightblue)}
+
+<===> named/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> srgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(srgb 0.45098 0.07843 0.823530))}
+
+<===> srgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> srgb-linear/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(srgb-linear 0.45098 0.07843 0.823530))}
+
+<===> srgb-linear/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> display-p3/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(display-p3 0.515 0.35 0.3 / 1))}
+
+<===> display-p3/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> a98-rgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(a98-rgb 0 1 0))}
+
+<===> a98-rgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> prophoto-rgb/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(prophoto-rgb 0.42444 0.934918 0.190922))}
+
+<===> prophoto-rgb/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> rec2020/input.scss
+@use "sass:color";
+a {b: color.is-legacy(?????)}
+
+<===> rec2020/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> xyz/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(xyz 0.0426 0.0442 0.0364))}
+
+<===> xyz/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> xyz-d50/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(xyz-d50 0.2005 0.14089 0.4472))}
+
+<===> xyz-d50/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> xyz-d65/input.scss
+@use "sass:color";
+a {b: color.is-legacy(color(xyz-d65 0.21661 0.14602 0.59452))}
+
+<===> xyz-d65/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> lab/input.scss
+@use "sass:color";
+a {b: color.is-legacy(lab(5 110 115))}
+
+<===> lab/output.css
+a {
+  b: false;
+}
+
+<===>
+================================================================================
+<===> lch/input.scss
```

Comment:
```suggestion
<===> named/input.scss
```
We could swap out one of the tests or add an additonal test called `named`. This will test that the function also works if the argument is explicitly named e.g. `$color: lch(90.6 52.8 197)`